main/src/ca/uwaterloo/flix/language/ast/ResolvedAst.scala

```diff
@@ -40,6 +41,10 @@ object ResolvedAst {
 
   case class LatticeOps(tpe: Type, bot: ResolvedAst.Expression, top: ResolvedAst.Expression, equ: ResolvedAst.Expression, leq: ResolvedAst.Expression, lub: ResolvedAst.Expression, glb: ResolvedAst.Expression, ns: Name.NName, loc: SourceLocation)
 
+  case class Class(sym: Symbol.ClassSym, tparam: ResolvedAst.TypeParam, signatures: List[ResolvedAst.Sig])
```

Comment:
If class goes before def, can you move this definition about def? I prefer to keep everything in some kind of order.